### PR TITLE
Jk/cumulus 3582

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-3582](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3582)

## Changes

Suggested CI changes re: 3582

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: packages/api/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "serve:debug": "LOCALSTACK_HOST=localhost LOCAL_ES_HOST=localhost NODE_ENV=test FAKE_AUTH=true node --inspect-brk ./bin/cli.js serve",
     "tsc": "../../node_modules/.bin/tsc",
     "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles",
-    "test": "../../node_modules/.bin/ava",
+    "test": "echo $PWD && for i in `find tests/ -name \\*.js -exec grep -l require\\(\\'ava\\'\\) {} \\;`; do echo TEST START: $i && ../../node_modules/.bin/ava $i; done;",
```
Comment (npauzenga) [inline packages/api/package.json:25]: So what is this change meant to capture? Were we missing some tests? Does it have something to do with the ordering we're messing with in the --streamed logs?

**Review by npauzenga — COMMENTED** [review]
Looks like a much better solution to better/untruncated logs. Just one question about the `test` script. I'm not sure what we're doing there.

File: package.json [hunk 2]
```diff
@@ -21,9 +21,9 @@
     "lint": "npm run lint-package-json && npm run eslint && lerna run python-lint && npm run lint-md",
     "lint-md": "markdownlint docs/**/*.md docs/*.md tf-modules/**/*.md tf-modules/*.md",
     "lint-package-json": "npmPkgJsonLint .",
-    "test": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test",
+    "test": "lerna run --ignore @cumulus/cumulus-integration-tests --stream --concurrency 1 test",
     "test:ci": "./scripts/run_ci_unit_coverage.sh",
-    "test:coverage": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test:coverage > testoutput.txt",
+    "test:coverage": "lerna run --ignore @cumulus/cumulus-integration-tests --stream --concurrency 1 test:coverage",
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply package.json:26]: It's only interleaved if you set concurrency > 1

File: packages/api/package.json (hunk 1 above)
Comment (Jkovarik) [reply packages/api/package.json:25]: It's intended to capture concretely which test AVA is running at a point in time.

Comment (npauzenga) [reply packages/api/package.json:25]: Ah ok so the task failed, the logs are truncated/insufficient and this is meant to tell us which test we're on, ideally around the time the task fails.

Yeah, hopefully redundant but I see it in the logs and it seems potentially helpful.

**Review by npauzenga — APPROVED** [review]
@Jkovarik I just pushed a commit (actually 2 including the main merge). We need to remove the `docker cp` command for the the artifact that no longer exists. I also removed the artifact from the Bamboo plan.

Otherwise, pending CI, I think this is good to go. Very hopeful this will be the ticket.

**Pull request merged**